.github/workflows/ci.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.19.x
+          go-version: 1.20.x
```

Comment:
Using new 1.20 only features, so have to drop support for old versions. Will need to cascade this change down to each connector.